definitions/features/foreman_server.rb

```diff
@@ -9,9 +9,11 @@ class ForemanServer < ForemanMaintain::Feature
       end
 
       def services
-        [
-          system_service('httpd', 30)
-        ]
+        if foreman_service_installed?
+          [system_service('foreman', 30)]
```

Comment:
We have introduced a `foreman.socket` that works like a service but does not need to be restarted. However, I think it would be helpful to ensure that the socket is running whenever we perform service operations. @ekohl what do you think regarding handling the socket?